docs/Userguide_python.rst

```diff
@@ -105,3 +105,44 @@ can be good to periodically clean up Conda's cache:
 .. prompt:: bash (<env>)$
 
    conda clean --all
+
+**Mamba**
+
+When installing new packages with ``conda install``, conda uses a built-in 
+dependency solver for solving the dependency graph of all packages (and their 
+versions) requested such that package dependency conflicts are avoided.
+
+In some cases, especially when there are many packages already installed in a 
+conda environment, conda's built-in dependency solver can struggle to solve the 
+dependency graph, taking several to tens of minutes, and sometimes never 
+solving. In these cases, it is recommended to try `libmamba 
```

Comment:
```suggestion
solving. In these cases, you can try the alternative solver `libmamba 
```

Reply:
If @obilaniu gives his seal of approval we could keep the previous form but not so long ago I think he had some reserve on the mass suggestion of mamba. this could have changed if mamba is now stable

Reply:
@satyaog We've discussed this in the original conversation, but in summary Mamba doesn't produce identical solves to Conda. Still, it has become more robust, and it's become the only practical solution in some cases.

As of the newest PyTorch 1.13.x's install instructions, any deviation from `-c pytorch -c nvidia` (such as by adding the incredibly common `-c conda-forge`) results in `conda create`/`conda install` taking _ridiculous_ amounts of time to come up with a solution. `mamba` on the other hand has no problems.

Definitely go for merge.